airflow-core/src/airflow/jobs/triggerer_job_runner.py

```diff
@@ -323,7 +330,7 @@ def start(  # type: ignore[override]
         proc = super().start(id=job.id, job=job, target=cls.run_in_process, logger=logger, **kwargs)
 
         msg = messages.StartTriggerer(requests_fd=proc._requests_fd)
-        proc._send(msg)
+        proc.stdin.write(msg.model_dump_json().encode("utf-8") + b"\n")
```

Comment:
```suggestion
        proc.stdin.write(msg.model_dump_json().encode() + b"\n")
```

Since `utf-8` is the default